packages/beacon-node/src/execution/engine/utils.ts

```diff
@@ -66,6 +66,7 @@ function getExecutionEngineStateForPayloadStatus(payloadStatus: ExecutionPayload
     case ExecutionPayloadStatus.INVALID:
     case ExecutionPayloadStatus.SYNCING:
     case ExecutionPayloadStatus.INVALID_BLOCK_HASH:
+    case ExecutionPayloadStatus.OPTIMISTIC_SYNC_DISABLED:
```

Comment:
In the case opt sync is disabled and EL returns `SYNCING` or `ACCEPTED`, payload status will be `OPTIMISTIC_SYNC_DISABLED`. 
Not sure if this will make EL status to be `SYNCED` or `SYNCING`. 

My gut says it should be `SYNCING` because the payload is not fully validated by EL. Happy to be corrected